.changeset/brave-gifts-occur.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
```

Comment:
We will bump the esbuild version [only in a minor](
https://developers.cloudflare.com/workers/wrangler/migration/update-v3-to-v4/#summary-of-changes).